components/eamxx/src/share/atm_process/IOPDataManager.cpp

```diff
@@ -234,8 +234,8 @@ initialize_iop_file(const util::TimeStamp& run_t0,
   else EKAT_ERROR_MSG("Error! No valid dimension for tsec in "+iop_file+".\n");
 
   // When we read vars, "time" must be treated as unlimited, to avoid issues
```

Comment:
Should this comment be updated since the function name is changing. Something like "When we read vars, the "time" dimension must be recorded to avoid issues"?